src/utils/isModelValueType.ts

```diff
@@ -1,10 +1,13 @@
+import isObjectLike from 'lodash/isObjectLike'
```

Comment:
Unused import here

Reply:
Nice catch - turns out we had a test set as `.only` too, which is also fixed now 🚀